runtime/crab/src/pallets/timestamp.rs

```diff
@@ -20,7 +20,7 @@
 use crate::*;
 
 impl pallet_timestamp::Config for Runtime {
-	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
+	type MinimumPeriod = ();
```

Comment:
What's the difference between these two values?

Reply:
Async requires this to be set to 0.